tests/installation/releasenotes.pm

```diff
@@ -62,10 +62,13 @@ sub run {
         push @no_relnotes, qw(base script desktop serverapp legacy sdk live);
         # WE has release-notes on SLE 15
         @no_relnotes = grep(!/^we$/, @no_relnotes);
-        # HA-GEO has been removed on SLE 15
-        @addons = grep(!/^geo$/, @addons);
     }
 
+    # No HA-GEO on ppc64le before SLE 15
+    # But HA-GEO in now (SLE 15+) included in HA extension for all platforms
+    # So we can remove it on ppc64le and SLE 15+
+    @addons = grep(!/^geo$/, @addons) if check_var('ARCH', 'ppc64le') or is_sle('15+');
+
```

Comment:
In other tests, `check_var('ARCH', 'ppc64le')` has been substituted for `get_var('OFW')`. Perhaps it would be better to also do this here. More info on https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4477

Reply:
Yes, I will change for OFW check, even if testing OFW is not the same as testing architecture, maybe we can have other architectures running OFW, but that has already been discussed in the PR you mentioned :)